dali/operators/sequence/optical_flow/optical_flow_test.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Nothing changed in this file as well 🤔

Reply:
reverted. and I swore to not care about automated header edits :(